tests/unit/sagemaker/config/test_config.py

```diff
@@ -236,3 +242,157 @@ def test_merge_of_s3_default_config_file_and_regular_config_file(
         s3_resource=s3_resource_mock,
     )
     del os.environ["SAGEMAKER_ADMIN_CONFIG_OVERRIDE"]
+
+
+def test_logging_when_overriden_admin_and_user_configs_are_found(get_data_dir, caplog):
```

Comment:
Can we parametrize these tests for different config inputs, specified or not specified?